conftest.py

```diff
@@ -93,7 +93,6 @@ class ArchiverSetup:
 
     def __init__(self):
         self.archiver = None
-        self.prefix = ""
```

Comment:
I removed all instances of 'prefix', which I believe is closer to your original desire. Since the prefix was essentially being used to indirectly check if the `archiver` was remote or not, I was considering just adding an `archiver` attribute called `is_remote` that is initialized to false but is set to true during the `remote_archiver` setup. 

Then, in the tests, instead of again indirectly checking if it is remote by looking into the `repository_location` as I have done here, I could instead use `archiver.is_remote` to check directly. Would this be a good approach?

Reply:
I also thought about something like `is_remote`,  but I think checking the `location` is better.

In future we might have different kinds of locations, not just local vs. ssh:// remote. In fact we already have that now with `socket://`.

Reply:
What if I added a `get_kind()` method to the `ArchiverSetup` class that returns the `kind` as a string, in a similar way that we name the `kinds` when generating tests. This would be a consistent approach, could lead to more readable code  then checking prefixes, and if we add different `kinds` later we could add them to the method easily.